hooks/useTarget/useTarget.tsx

```diff
@@ -84,7 +84,9 @@ export const TargetProvider: FunctionComponent = ({ children }) => {
         timestamp: timestampResult.timestamp - BLOCKS_IN_A_DAY * 12,
       };
     }
-    setResult(newValue);
+    if (newValue.now && newValue.yesterday) {
```

Comment:
instead of this, might be better to merge the old value with the new stuff. So like:

```ts
const newValue: TargetLookup = { ...result };
```

Reply:
and maybe short-circuit if there are no new values

Reply:
I think the real issue here is that urql isn't holding the old value, once it starts fetching a new one it goes undefined

Reply:
@cnasc so what i'm worried about with this, is now the effect has a dep on `result` and won't this result in non-stop updating?

Reply:
Sorry, should have done this as an example instead.
```js
setResult(prev => { /* do stuff with previous value here */ });
```
Yes, meant wagmi